gradlew

```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 
 #
-# Copyright � 2015-2021 the original authors.
+# Copyright � 2015-2021 the original authors.
```

Comment:
Seems that there is a codification problem with this file

Reply:
Iiek! Fixed